README.md

```diff
@@ -1,2 +1,25 @@
+
 # MotivateU
-A desktop application that gives random motivational quotes
+
+https://img.shields.io/github/license/Shankhanil/MotivateU
```

Comment:
There is an issue regarding this line. The link is supposed to be rendered into a label, which is not happening. I will try and fix that ASAP